daemon/container_unix.go

```diff
@@ -896,8 +900,6 @@ func (container *Container) configureNetwork(networkName, service, networkDriver
 		}
 	}
 
-	container.updateSandboxNetworkSettings(sb)
```

Comment:
I'm not necessarily an expert on libnetwork, but looking at the code, I think @aboch is correct, in that there is sort of some term overloading here of the use of "Key"; setExternalKey operations are not actually setting the sb.Key(), which remains a short variant of the container ID.

I think this call can/should stay, as I don't see where these values are going to be set on the `container.NetworkSettings` anywhere else?